### PR TITLE
Update community.yml

### PR DESCRIPTION
**Pull request by muthuannamalai12**
I have updated the issue-message, Pr-Message with new ones, and added a footer in Community.yml

### CODE REVIEW

File: .github/workflows/community.yml [hunk 1]
```diff
@@ -19,8 +19,9 @@ jobs:
       - uses: EddieJaoudeCommunity/gh-action-community/src/welcome@main
```
Comment (eddiejaoude) [inline .github/workflows/community.yml]: Could you update this to `EddieHubCommunity` too

File: .github/workflows/community.yml [hunk 2]
```diff
@@ -16,16 +16,17 @@ jobs:
...
-      - uses: EddieJaoudeCommunity/gh-action-community/src/welcome@main
+      - uses: EddieHubCommunity/gh-action-community/src/welcome@main
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
-          issue-message: "<h1>It's great having you contribute to this project</h1> Feel free to raise an <strong>Issue</strong>! Welcome to the community :nerd_face:"
-          pr-message: "<h1>It's great having you contribute to this project</h1> Feel free to create a <strong>Pull Request</strong>! Welcome to the community :nerd_face:"
+          issue-message: '<h1>It''s great having you contribute to this project</h1> Welcome to the community :nerd_face:'
+          pr-message: '<h1>It''s great having you contribute to this project</h1> Welcome to the community :nerd_face:'
+          footer: 'If you would like to continue contributing to open source and would like to do it with an awesome inclusive community, you should join our <a href="http://discord.eddiehub.org">Discord</a> chat and our <a href="http://github.eddiehub.org">GitHub Organisation</a> - we help and encourage each other to contribute to open source little and often 🤓 . Any questions let us know.'
```
Comment (Vyvy-vi) [inline .github/workflows/community.yml]: Could there be this small consistency added to this?:
```suggestion
          footer: 'If you would like to continue contributing to open source and would like to do it with an awesome inclusive community, you should join our <a href="http://discord.eddiehub.org">Discord Server</a> and our <a href="http://github.eddiehub.org">GitHub Organisation</a> - we help and encourage each other to contribute to open source little and often 🤓 . Any questions let us know.'
```

**Review by schmelto — APPROVED** [review]

**Review by Vyvy-vi — APPROVED** [review]
LGTM-!

**Review by galahad42 — APPROVED** [review]

**Review by eddiejaoude — APPROVED** [review]
Great team work 👍

**Pull request merged**